index.html

```diff
@@ -8,7 +8,23 @@
     <title>Awesome Books</title>
 </head>
 <body>
+    <header class="nav-container">
+        <div class="logo">
+          <a href="">Awesome Books </a>
+        </div>
+        <nav class="menu">
+          <ul class="nav-list">
+            <li class="list">
+              <a href="" class="list">Book List</a>
+            </li>
+            <li class="list"><a href="" class="add">Add Books</a></li>
+            <li class="list"><a href="" class="contact-me">Contact</a></li>
```

Comment:
Great design @krisbobo 🤝

- [ ] Kindly make sure that when you click on the `add-book` navigation, the `add-book-form` displays, and also when you click on the `contact`, the contact information displays. Right now nothing is rendered for both add book and contact navigations.
![add book](https://user-images.githubusercontent.com/63917725/180620020-7ea4c205-ffc6-42cb-a1d9-77582bfc222d.PNG)